fix: [ANDROAPP-3719] Completion percentage does take into consideration unsupported fields

ferdyrod opened the pull request:
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ANDROAPP-3719](https://jira.dhis2.org/browse/ANDROAPP-3719)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

andresmr reviewed (approved):
Good job, a couple of questions: 

- So we have removed to tap in the completion spinner and see a summary right?.
- The completions spinner only works for events? not for enrolment?

ferdyrod commented:
> Good job, a couple of questions:
> 
> * So we have removed to tap in the completion spinner and see a summary right?.
> * The completions spinner only works for events? not for enrolment?

1. The tap to go to the summary was removed before this changes, I just removed the unused usecase.
2. Yes, it's only present in events.
